docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -1,8 +1,8 @@
 ---
-sidebar_label: 'Deploy a Custom Pack'
-title: 'Deploy a Custom Pack'
-description: 'How to create and deploy a custom pack using the manifest files or Helm charts in Spectro Cloud.'
-icon: ''
+sidebar_label: "Deploy a Custom Pack"
+title: "Deploy a Custom Pack"
+description: "Learn how to deploy applications to a Kubernetes cluster with custom add-on packs. Deploy your custom pack from the Spectro Registry or an OCI registry. Learn how to get started with Palette’s custom packs and reuse them in multiple deployments."
+icon: ""
 hide_table_of_contents: false
 toc_min_heading_level: 2
 toc_max_heading_level: 2
```

Comment:
Are these required? You have quite a few H3 headings

Reply:
Oh, that's true! Thanks for catching this.

Reply:
Actually, I need to keep it as it is because I am using tabs in the **Create a Cluster Profile** section, and the headings got messy.